sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/batch/implementation/BatchAsyncBatcherExecutor.java

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.batch.implementation;
+
+import reactor.core.publisher.Mono;
+
+/**
+ * Executor implementation that processes a list of operations.
+ */
+interface BatchAsyncBatcherExecutor {
+    Mono<PartitionKeyRangeBatchExecutionResult> apply(PartitionKeyRangeServerBatchRequest request) throws Exception;
```

Comment:
this shouldn't throw, should instead return a Mono of error.

Reply:
Yeah. Fixed. Thanks.